api/managementpb/mysql.proto

```diff
@@ -90,13 +90,13 @@ message AddMySQLResponse {
 
 // MySQL service provides public Management API methods for MySQL Service.
 service MySQL {
-  // Add MySQL adds MySQL Service and starts several Agents.
+  // AddMySQL adds MySQL Service and starts several Agents.
   // It automatically adds a service to inventory, which is running on provided "node_id",
   // then adds "mysqld_exporter", and "qan_mysql_perfschema" agents
   // with provided "pmm_agent_id" and other parameters.
-  rpc Add(AddMySQLRequest) returns (AddMySQLResponse) {
+  rpc AddMySQL(AddMySQLRequest) returns (AddMySQLResponse) {
     option (google.api.http) = {
-      post: "/v1/management/MySQL/Add"
+      post: "/v1/management/MySQL/AddMySQL"
```

Comment:
URL should end with `/Add`. For example:
```
service Nodes {
  rpc GetNode(GetNodeRequest) returns (GetNodeResponse) {
    option (google.api.http) = {
      post: "/v1/inventory/Nodes/Get"
      body: "*"
    };
  }
```

We have "Node" in method/request/response mostly to avoid name clashes, but in URL we don't have this problem.

Reply:
Another example:
```
  rpc AddGenericNode(AddGenericNodeRequest) returns (AddGenericNodeResponse) {
    option (google.api.http) = {
      post: "/v1/inventory/Nodes/AddGeneric"
      body: "*"
    };
  }
```

Reply:
Wait for a minute https://github.com/percona/pmm/pull/341#discussion_r279621836 - is your previous comment only about MongoDB?

Reply:
fixed